dependencies.toml

```diff
@@ -306,6 +309,10 @@ version.ref = "slf4j"
 module = "org.slf4j:log4j-over-slf4j"
 version.ref = "slf4j"
 
+[libraries.snappy]
+module = "org.xerial.snappy:snappy-java"
```

Comment:
Add this as `implementation` dependency of `:server` module?

Reply:
Yes, 😓